packages/components/breadcrumbs/src/breadcrumbs.ts

```diff
@@ -126,30 +127,37 @@ export class Breadcrumbs extends ScopedElementsMixin(LitElement) {
       ${this.noHome
         ? nothing
         : html`
-            <a href=${this.homeUrl}><sl-icon name="house"></sl-icon>${isMobile() ? '' : msg('Home')}</a>
-            <sl-icon name="chevron-right"></sl-icon>
+            <a href=${this.homeUrl}><sl-icon name="home-blank"></sl-icon>${isMobile() ? '' : msg('Home')}</a>
+            <sl-icon name="slash-forward"></sl-icon>
           `}
       ${this.breadcrumbs.length > this.collapseThreshold
         ? html`
-            <sl-menu-button fill="link">
-              <sl-icon name="ellipsis" slot="button"></sl-icon>
+            <sl-button fill="link" id="button" @click=${this.#onClick} icon-only aria-label=${msg('More breadcrumbs')}>
```

Comment:
`icon-only` is an internal thing. I don't think we should set that here. Also not necessary?

Reply:
🙈 apparently not all Max component behaviours are out of my system... We needed to add this manually :D